src/main/java/org/folio/marccat/resources/domain/FixedField.java

```diff
@@ -1787,7 +1844,8 @@ public String getSerialOriginalAlphabetOfTitleCode() {
   }
 
   /**
-   * @param serialOriginalAlphabetOfTitleCode The serialOriginalAlphabetOfTitleCode
+   * @param serialOriginalAlphabetOfTitleCode The
```

Comment:
Put it back on the same line:
 * @param serialOriginalAlphabetOfTitleCode The serialOriginalAlphabetOfTitleCode